src/App.js

```diff
@@ -3,20 +3,18 @@ import './App.scss';
 
 import todos from './api/todos';
 import users from './api/users';
+import { TodoList } from './components/TodoList/TodoList';
+
+const preparedTodos = todos.map(task => ({
+  ...task,
+  user: { ...users.find(user => user.id === task.userId) },
```

Comment:
```suggestion
  user: users.find(user => user.id === task.userId),
```
you can leave link to the original object